WordPress/Classes/ViewRelated/Post/PostSettingsViewController+Swift.swift

```diff
@@ -223,10 +223,7 @@ extension PostSettingsViewController {
             if let index = pages.firstIndex(of: page) {
                 pages = pages.remove(from: index)
             }
-            let viewController = ParentPageSettingsViewController.make(with: pages, selectedPage: page) { [weak self] in
-                self?.navigationController?.popViewController(animated: true)
-                self?.tableView.reloadData()
-            }
+            let viewController = ParentPageSettingsViewController.make(with: pages, selectedPage: page)
```

Comment:
The production behavior is that it shows you your selection, and then you need to tap "back" manually, like in most other details screen in settings.